python/ray/data/datasource/file_meta_provider.py

```diff
@@ -329,17 +329,17 @@ def prefetch_file_metadata(
             return _fetch_metadata(pieces)
 
 
-def _handle_read_s3_files_error(error: OSError, paths: Union[str, List[str]]) -> str:
-    # Specially handle AWS error when reading files, to give a clearer error message
-    # to avoid confusing users. The real issue is most likely that the AWS S3 file
-    # credentials have not been properly configured yet.
+def _handle_read_os_error(error: OSError, paths: Union[str, List[str]]) -> str:
```

Comment:
Add a quick note to make it explicit that this is not comprehensive yet, and should be extended as more errors arise?

Reply:
@matthewdeng - sure, added.